src/Abp.AutoMapper/AutoMapper/AutoMapperObjectMapper.cs

```diff
@@ -6,7 +6,7 @@ namespace Abp.AutoMapper
 {
     public class AutoMapperObjectMapper : IObjectMapper
     {
-        private readonly IMapper _mapper;
+        protected readonly IMapper _mapper;
```

Comment:
Nonetheless, if it is useful for developers, I think we still can lift the restriction here.

Please rename _mapper to Mapper as we use PascalCase for protected property

Reply:
Done

Reply:
You will need to rename usage of _mapper as well.